app/templates/admin/extras/donations_change_form.html

```diff
@@ -3,6 +3,10 @@
 
 {% block submit_buttons_bottom %}
   {{ block.super }}
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.13.1/themes/base/jquery-ui.css">
+  <script src="http://code.jquery.com/jquery-1.9.1.js"></script>
```

Comment:
we should already have jquery. please see https://github.com/CodeTheChangeUBC/reBOOT/blob/38e9c9067aacd7fd8b78c7f904282f94fc1e3c18/app/templates/admin/base_site.html#L7